EKF/RingBuffer.h

```diff
@@ -86,12 +88,14 @@ class RingBuffer {
 		return true;
 	}
 
-	void unallocate() {
+	void unallocate()
+	{
```

Comment:
This one actually stands out as being different than PX4.

Reply:
From the google style guide.

 - "The open curly brace is always on the end of the last line of the function declaration, not the start of the next line."
 - "The close curly brace is either on the last line by itself or on the same line as the open curly brac"

Reply:
Ok, you got me. But let me add an empty line after the function header.